### PR TITLE
add clusterbuster release index

### PR DESCRIPTION
**Pull request by ebattat**
This PR contains:
1. Add dedicate release index for clusterbuster
2. Add release run_type
3. Add clusterbuster new version v1.1.3-kata-ci using uuid

### CODE REVIEW

**Review by RobertKrawitz — COMMENTED** [review]
Just want to understand how delete_index_content is going to be used; otherwise I'm good with this.

File: benchmark_runner/common/elasticsearch/elasticsearch_operations.py [hunk 1]
```diff
@@ -199,3 +199,13 @@ def get_elasticsearch_index_by_id(self, index: str, id: str):
...
+
+    @typechecked()
+    @logger_time_stamp
+    def delete_index_content(self, index: str):
+        """
+        Caution: This method delete all index content
+        :param index: index name
+        :return:
+        """
+        return self.__es.indices.delete(index=index, ignore=[400, 404])
```
[the remark this answers is not in the document]
Comment (ebattat) [reply benchmark_runner/common/elasticsearch/elasticsearch_operations.py:211]: For that we need a uuid.
We need to fetch index according uuid and get all the related ids, and then delete it.
I did a fast cleanup and delete the whole index content

[the remark this answers is not in the document]
Comment (RobertKrawitz) [reply benchmark_runner/common/elasticsearch/elasticsearch_operations.py:211]: Then the parameter should be named 'uuid'.

**Review by RobertKrawitz — APPROVED** [review]
lgtm

**Pull request merged**